scripts/dts/extract_dts_includes.py

```diff
@@ -441,7 +441,9 @@ def yaml_inc_error(msg):
 def generate_defines():
     # Generates #defines (and .conf file values) from DTS
 
-    for node_path in reduced.keys():
+    # sorted() otherwise Python < 3.6 randomizes the order of the flash
+    # partition table
+    for node_path in sorted(reduced.keys()):
```

Comment:
Looks like I missed this earlier, but could get rid of the `.keys()`. `x in dict`, `for x in dict`, `sorted(dict)`, etc., always uses the keys.

Reply:
Agreed but I'd like to keep this minimal fix as minimal as possible :-) Not just for aesthetic reasons but also cherry-picking for testing / backporting etc.

Reply:
Alright, fine with me.